debian/source/options

```diff
@@ -0,0 +1 @@
+tar-ignore = ".git/*"
```

Comment:
Should probably do `.git*` entirely, to avoid `.gitignore` and such too.

Reply:
nvllsvm has expressed desire for explicit ignores so we don't accidentally start ignoring things we want.

See https://github.com/jellyfin/jellyfin/pull/392#discussion_r245174893

Reply:
whichever is fine with me, just reach a consensus and I'll see it through :)

Reply:
Ok, so after some pondering I think it's good to just remove .git. To largely leave the repo intact seems like the best route to me..

Reply:
In this case, explicit ignore would be
```
/.git/*
.gitignore
```

Reply:
I'll try with /.git/* but from the man page of dpkg-source I don't think I can have multiple regexes, just one big one. And I think it's common knowledge how big regexes are hard to maintain

Reply:
This also wouldn't ignore submodule git. I think Debian packaging is different from Docker, and he was referring mostly to the Markdownf files anyways. cc @nvllsvm for clarification.

I'd like to completely exclude *all* `git`-related files from the packages if at all possible, and then in the source package do what a lot of source packages do, that packaging is done in git at "url". e.g.

```
$ apt-get source ffmpeg
Reading package lists... Done
NOTICE: 'ffmpeg' packaging is maintained in the 'Git' version control system at:
https://salsa.debian.org/multimedia-team/ffmpeg.git
Please use:
git clone https://salsa.debian.org/multimedia-team/ffmpeg.git
to retrieve the latest (possibly unreleased) updates to the package.
Need to get 8,944 kB of source archives.
```

Reply:
It looks like '.git' removes files / directories named .git, while .git/* removes contents of a directory (leaving an empty directory) and that since the base path is unknown, this is the best I can do with that option..

Reply:
Leaving the empty dir is messy, `.git*` accomplishes the same thing, and I think that's fine. I've been testing with `.git*` and it generates a perfectly reasonable source package that's 361MB, versus the >1G source package with the history. The result builds fine as a standalone Debian source package, so I think we're OK with that format.

Reply:
That is to say, I'm not sure if there's any subdirectories it's also removing the `.git` dir from, but if it is that's ideal too.